src/create-default-events.js

```diff
@@ -11,8 +11,9 @@ export default function createDefaultEvents (editable) {
      * @event focus
      * @param {HTMLElement} element The element triggering the event.
      */
-    focus (element) {
+    focus (element, selection) {
       behavior.focus(element)
+      behavior.selection(element, selection)
```

Comment:
Maybe we could add logic to determine if the `selection` has actually changed so we don't emit every selection event twice after this change.

What do you think?